src/site/_includes/macros/show-card.njk

```diff
@@ -0,0 +1,42 @@
+{#
+  @typedef {ShowsItem} show - The show to display in card format.
+#}
+{% macro showCard(show) %}
+
+{% set data = show.data %}
+{% set title = data.title %}
+{% set description = data.subhead %}
+{% set url = show.url %}
+{% set hero = data.thumbnail or null %}
+
+<div class="w-card" role="listitem">
+  <article class="w-card-base {{ 'w-card-base__with--cover' if hero }}">
+    {% if hero %}
+      <div class="w-card-base__cover">
+        {#
+          We intentionally hide the hero image from screen readers because it appears out of context
+          from the rest of the article content and may be confusing. Since the link for the article is duplicated
+          in the heading down below, this might be an OK tradeoff.
+        #}
+        <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
+          <figure>
```

Comment:
Don't wrap this in a figure, it adds extra margin that makes the image too small.